site/cypress/e2e/Cypress-UI/navigation_page.spec.js

```diff
@@ -9,6 +7,91 @@ function validate_navigation_page_sections(sections) {
     });
 }
 
+const white = 'rgb(255, 255, 255)';
+const blue = 'rgb(0, 99, 152)';
+const red = 'rgb(220, 53, 69)';
+const green = 'rgb(95, 183, 96)';
+const buttonOrder = ['team-btn', 'submit-btn', 'grade-btn', 'quick-link-btn'];
+
+function checkButtons(courseID, buttonText, buttonColor) {
+    // assert that the course does not exist
+    if (buttonText === null && buttonColor === null) {
+        cy.get(`[data-testid="${courseID}"]`).should('not.exist');
+    }
+    else {
+        for (let i = 0; i < buttonOrder.length; i++) {
+            if (buttonText[i]) {
+                cy.get(`[data-testid="${courseID}"]`).find(`[data-testid="${buttonOrder[i]}"]`).contains(buttonText[i]);
```

Comment:
```suggestion
                cy.get(`[data-testid="${courseID}"]`).find(`[data-testid="${buttonOrder[i]}"]`).should('contain', buttonText[i]);
```

It's better to use assertions in this case. `.contains(x)` should be use to find an element that contains the given parameter, and then do something with it.